comeetTests/Data/Fetcher/EnvironmentTests.swift

```diff
@@ -27,7 +27,7 @@ class EnvironmentTests: XCTestCase {
     }
     
     func testGetRoomsProduction() {
-        let environment: Environment = .Production
+        let environment: Environment = .Mocked
```

Comment:
Why are we going back to the mocked environment??? We should not do this. If you need to have a toggle to change envs let's use the settings bundle: https://github.com/bookaroom/comeet-iOS/tree/master/comeet/Settings.bundle